codegen/templates/dependency_struct.tmpl

```diff
@@ -0,0 +1,24 @@
+{{$instance := . -}}
+package {{$instance.PackageInfo.PackageName}}
+
+import (
+	{{range $classType, $moduleInstances := $instance.ResolvedDependencies -}}
+	{{range $idx, $moduleInstance := $moduleInstances -}}
+	{{$moduleInstance.PackageInfo.ImportPackageAlias}} "{{$moduleInstance.PackageInfo.ImportPackagePath}}"
+	{{end -}}
+	{{end -}}
+)
+
+type {{$instance.PackageInfo.ExportType}}Deps struct {
+	{{range $classType, $moduleInstances := $instance.ResolvedDependencies -}}
+	{{$classType | pascal}} {{$instance.PackageInfo.ExportType}}{{$classType | pascal}}Deps
```

Comment:
This reminds me of `minute := 60; hour:= 60 * minute; day := 24 * hours; year := 365 * day;`